docs/getting-started/supported-languages-and-tools.md

```diff
@@ -176,15 +188,18 @@ The table below lists all languages that Codacy supports and the corresponding t
       <td>-</td>
       <td>-</td>
     </tr>
+    <!--
```

Comment:
to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools